README.md

```diff
@@ -1,110 +1,148 @@
-
+# Convert to Tokyo time
+tokyo_time = utc_time.astimezone(ZoneInfo('Asia/Tokyo'))
 
-def get_response() -> ApiResponse:
-    response = requests.get('https://some-api.com/time')
-    return ApiResponse.parse_obj(response.json())
+# Convert to New York time
+ny_time = utc_time.astimezone(ZoneInfo('America/New_York'))
 ```
 
-The returned `ApiResponse` instance is guaranteed to have parsed the `current_time` attribute 
-as UTC+0 no matter how the api provided the timestamp. If no timezone information is provided, 
-it will be assumed to be UTC+0.
+### Handling naive datetimes
 
-Expanding the module can be done with little effort, by creating a new class that inherits `datetime_tz`:
+```python
+from heliclockter import datetime_utc, datetime_tz
+from datetime import datetime
+
+# datetime_utc assumes UTC for naive inputs
+naive_dt = datetime(2022, 11, 4, 15, 30, 0)
+utc_dt = datetime_utc.from_datetime(naive_dt)  # OK - assumes UTC
+
+# datetime_tz requires explicit timezone
+try:
+    tz_dt = datetime_tz.from_datetime(naive_dt)  # Raises error
+except Exception as e:
+    print(e)  # "Cannot create aware datetime from naive if no tz is assumed"
+```
+
+### Custom timezone classes
 
 ```python
 from zoneinfo import ZoneInfo
 from heliclockter import datetime_tz
 
-
 class datetime_cet(datetime_tz):
-    """
-    A `datetime_cet` is a `datetime_tz` but which is guaranteed to be in the 'CET' timezone.
-    """
-
+    """Datetime guaranteed to be in CET timezone."""
     assumed_timezone_for_timezone_naive_input = ZoneInfo('CET')
+
+# Parse naive timestamps as CET
+aware_dt = datetime_cet.strptime('2022-11-04T15:49:29', '%Y-%m-%dT%H:%M:%S')
 ```
 
-If you have a timestamp which is naive, *but* the timezone in which it is made is known to you,
-you can easily create a `datetime_tz` instance using your own defined classes:
+### Type safety with mypy
 
 ```python
-aware_dt = datetime_cet.strptime('2022-11-04T15:49:29', '%Y-%m-%dT%H:%M:%S')
-# datetime_cet(2022, 11, 4, 15, 49, 29, tzinfo=zoneinfo.ZoneInfo(key='CET'))
+from heliclockter import datetime_utc, datetime_local
+
+def schedule_task(when: datetime_utc) -> None:
+    """Schedule a task at a specific UTC time."""
+    print(f"Task scheduled for {when.isoformat()}")
+
+# Type checker ensures only UTC datetimes are passed
+utc_time = datetime_utc.now()
+schedule_task(utc_time)  # ✓ OK
+
+local_time = datetime_local.now()  
+schedule_task(local_time)  # ✗ Type error
 ```
 
-About the name
---------------
+## API Overview
+
+### Core Classes
+
+- **`datetime_tz`** - Base class for timezone-aware datetimes
+- **`datetime_utc`** - Always UTC (naive inputs assumed UTC)
+- **`datetime_local`** - Always local timezone (naive inputs assumed local)
+
+### Key Methods
+
+- `now()` - Current time
+- `from_datetime()` - Convert from standard datetime
+- `strptime()` - Parse string to datetime
+- `future()/past()` - Create relative timestamps
+- `astimezone()` - Convert between timezones
```

Comment:
`astimezone` should not be in this list as it doesn't actually work at runtime. We could try to make it work, but I expect it to be tricky with the type hinting as any arbitrary custom class definition could be the return type.